mizrahi2021re/openreview/packages.tex

```diff
@@ -0,0 +1,47 @@
+ % if you need to pass options to natbib, use, e.g.:
+%     \PassOptionsToPackage{numbers, compress}{natbib}
+% before loading neurips_2019
+
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[final]{neurips_2019} % Remove for ReScience
+
+% to avoid loading the natbib package, add option nonatbib:
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```

Comment:
Please remove this import `fontenc` as this appears to break ReScience fonts (check #4 for discussions)